x-pack/plugins/ml/public/application/timeseriesexplorer/timeseriesexplorer_embeddable_chart/timeseriesexplorer_embeddable_chart.js

```diff
@@ -0,0 +1,931 @@
+                            values: { functionLabel: chartDetails.functionLabel },
+                          }
+                        )}
+                      </span>
+                      &nbsp;
+                      {chartDetails.entityData.count === 1 && (
+                        <EuiTextColor color={'success'} size={'s'} component={'span'}>
+                          {chartDetails.entityData.entities.length > 0 && '('}
+                          {chartDetails.entityData.entities
+                            .map((entity) => {
+                              return `${entity.fieldName}: ${entity.fieldValue}`;
+                            })
+                            .join(', ')}
+                          {chartDetails.entityData.entities.length > 0 && ')'}
+                        </EuiTextColor>
+                      )}
+                      {chartDetails.entityData.count !== 1 && (
+                        <EuiTextColor color={'success'} size={'s'} component={'span'}>
+                          {chartDetails.entityData.entities.map((countData, i) => {
+                            return (
+                              <Fragment key={countData.fieldName}>
+                                {i18n.translate(
+                                  'xpack.ml.timeSeriesExplorer.countDataInChartDetailsDescription',
+                                  {
+                                    defaultMessage:
+                                      '{openBrace}{cardinalityValue} distinct {fieldName} {cardinality, plural, one {} other { values}}{closeBrace}',
+                                    values: {
+                                      openBrace: i === 0 ? '(' : '',
+                                      closeBrace:
+                                        i === chartDetails.entityData.entities.length - 1
+                                          ? ')'
+                                          : '',
+                                      cardinalityValue:
+                                        countData.cardinality === 0
+                                          ? allValuesLabel
+                                          : countData.cardinality,
+                                      cardinality: countData.cardinality,
+                                      fieldName: countData.fieldName,
+                                    },
+                                  }
+                                )}
+                                {i !== chartDetails.entityData.entities.length - 1 ? ', ' : ''}
+                              </Fragment>
+                            );
+                          })}
+                        </EuiTextColor>
+                      )}
+                    </h2>
+                  </EuiTitle>
+                </EuiFlexItem>
+
+                <EuiFlexItem grow={false}>
+                  <TimeSeriesExplorerHelpPopover />
```

Comment:
Note that if you remove the drag select for creating annotations, then the section in the help text about adding annotations will need hiding.

Reply:
Removed drag selection in c5304be32535efc1a303a7c6654e6423326c13a8 and removed annotations in 85d0042cfd1e54a04be691c2935636e5bc447af3